reana_server/rest/launch.py

```diff
@@ -104,13 +104,22 @@ def launch(user, url, name="", parameters="{}", specification=None):
             Request succeeded. Information of the workflow launched.
           schema:
             type: object
+            required:
+              - workflow_id
+              - workflow_name
+              - message
             properties:
               workflow_id:
                 type: string
               workflow_name:
                 type: string
               message:
                 type: string
+              validation_warnings:
+                description: >-
+                    Dictionary of validation warnings, if any. Each
+                    key is a property that was not correctly validated.
+                type: object
```

Comment:
What do you think about specifying this more, so that it is easily usable also in r-client-go? With the current specification, `validation_warnings` is a `interface{}` (note that currently we do not use the launch endpoint in the clients, but I think it's still a good idea to have precise OpenAPI specs)